nixos/modules/programs/ghidra.nix

```
@@ -0,0 +1,47 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.ghidra;
+in
+{
+  options.programs.ghidra = {
+    enable = lib.mkEnableOption "Ghidra, a software reverse engineering (SRE) suite of tools";
+
+    gdb = lib.mkOption {
+      default = false;
```

Since this is the only thing this module does, should not just make it the default?

```suggestion
      default = true;
```

---

fair enough, thanks! I applied the diff